### DISCUSSION TITLE
Cron Jobs

### DISCUSSION THREAD

**Discussion by bryiac** [planetscale/discussion#265] [Feature Requests] [answered]
Title explains it all but ill explain further 👍 

Would be cool to have a little section within the dashboard to provide queries that get ran at the set interval would be useful for cleaning stuff up among other things

**Comment by orware** [comment, marked as answer]
@coloredwax Thank you sharing this request!

I've actually received something similar about two weeks ago from one of our users so I'll add a link to this discussion there in our internal issue tracking that feature request.

**Reply by binjamil** [reply]
Is there any update on this feature request?

**Reply by runmaxde** [reply]
Any updates here? ✋ 
THX in advance

**Reply by mscoutermarsh** [reply]
Hey @runmaxde. We have no plans in the near future to build this into PlanetScale
